draft-looker-oauth-attested-key-based-client-authentication.md

```diff
@@ -94,34 +94,51 @@ The following diagram depicts the conceptual interactions.
                      +------->|               |<------------------------>|               |
   (1) Generate       |        |    Client     |    (6) Interaction       | Authorization |
   Attestation Key    |        |   Instance    |    using Client          |    Server     |
-                     +--------|               |    Key Attestation + PoP |               |
+                     +--------|               |    Attestation + PoP     |               |
                               |               |    for authentication    |               |
                               +---------------+                          +---------------+
                                  / \      |
                                   |       |
                                   +-------+
                                 (5) Generate
-                              Key Attestation PoP
+                           Client Attestation PoP
 ~~~
 
-Note defining the protocol for steps 2 and 4, including how the client instance authenticates with the client backend is out of scope of this specification.
+At first, the Authorization Server will communicate its requirements for the client authentication through its metadata. A Client instance that wants to request an access token from this Authorization Server will need to obtain a Client Attestation JWT, for example, from its client backend first.
 
-This specification only defines the format of the client assertion that a client instance uses to authenticate in its interactions with an authorization server (indicated in step 6), which is comprised of two key parts:
+Therefore, the client will generate a key (Client Instance Key) and (platform specific) attestations to prove its authenticity and security to the client backend (step 1). The Client instance sends this data to the client backend in request for a Client Attestation JWT (step 2). If the Client Backend successfully validates the Client Instance Key and further data, it will generate a signed Client Attestation JWT (step 3). As the Client Attestation JWT is cryptographically bound to the Client Instance Key generated by the client, the attestation is bound to this particular client instance. The client backend will respond to the client's request by sending the Client Attestation JWT (step 4).
```

Comment:
I'd not use the future, I think that would be better talking in the present

Reply:
`client instance` -> `Client instance`

Reply:
is client backend a defined term? If yes, we should uppercase it accordingly

Reply:
I will capitalize things later once they have an entry in the terminology